release/check_wheels.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+# Authors: Michael Penkov <m@penkov.dev>
+# Copyright (C) 2019 RaRe Technologies s.r.o.
```

Comment:
```suggestion
# Copyright (C) 2019 Radim Rehurek <radimrehurek@seznam.cz>
```

Reply:
Any reasons for "copyright header" in each file? I just want to understand why this needed